boards/arm/nrf5340_audio_dk_nrf5340/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2021 Nordic Semiconductor ASA.
```

Comment:
Minor, but you could consider adding the current year for the copyright notices in these files, e.g. as
```suggestion
# Copyright (c) 2021-2022 Nordic Semiconductor ASA.
```

But that's optional